lib/widgets.js

```diff
@@ -43,6 +44,66 @@ var input = function (type) {
     };
 };
 
+var choiceValueEquals = function (value1, value2) {
+    return !is.array(value1) && !is.array(value2) && String(value1) === String(value2);
+};
+
+var isSelected = function (value, choice) {
+    return value && (is.array(value) ? value.some(choiceValueEquals.bind(null, choice)) : choiceValueEquals(value, choice));
```

Comment:
I've fixed the `Array.isArray` uses in 332580b91b9a27b7617650edb1f574a4227540f5 and the `Object.keys` ones in 571d84c0dccd25b45e522374a209e8384d05bdc3 - I think we need to use a dep instead of using `some` or `reduce`, but since `ag '\.(some|reduce|filter|map|every|find|findIndex|reduceRight|includes)'` yields me about 12-13 uses in master already, it can wait until after this is merged.